README.md

```diff
@@ -120,14 +120,15 @@ $ python3 python_backend/examples/add_sub/client.py
 * numpy
 * rapidjson-dev
 * libarchive-dev
+* zlib1g-dev
 
 ```
 pip3 install numpy
 ```
 
-On Ubuntu or Debian you can use the command below to install `rapidjson` and `libarchive`:
+On Ubuntu or Debian you can use the command below to install `rapidjson`, `libarchive`, and `zlib`:
```

Comment:
Remove extra comma after libarchive

Reply:
It is intentional. See below:

https://en.wikipedia.org/wiki/Serial_comma